src/components/AddItem/WSRTheme.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { ThemeInterface } from 'wix-style-react/dist/src/ThemeProvider';
+
+import { ReactComponent as PlusXL } from '../../assets/icons/PlusXL.svg';
+import { ReactComponent as PlusL } from '../../assets/icons/PlusL.svg';
+import { ReactComponent as PlusM } from '../../assets/icons/plus.svg';
+import { ReactComponent as PlusS } from '../../assets/icons/PlusS.svg';
+
+export const theme = (className): ThemeInterface => ({
+  className,
+  icons: {
+    AddItemButton: {
+      tiny: PlusS,
+      small: PlusM,
+      medium: PlusL,
+      large: PlusXL,
+      image: null,
```

Comment:
@sivanharel I renamed this file to `*.tsx` and also added the `image ` field since it's required in https://github.com/wix/wix-style-react/blob/master/src/ThemeProvider/test/ThemeProvider.tsx#L39...

Maybe we should make it optional there or remove it if it's irrelevant?